certg.py

```diff
@@ -67,7 +67,8 @@ def main():
         # Below you need to add the route to the executable of inkscape
         # e.g Windows: C:/Program Files/Inkscape/inkscape.exe
         # e.g Mac: /Applications/Inkscape.app/Contents/MacOS/inkscape
-        cmd = ["/Applications/Inkscape.app/Contents/MacOS/inkscape", '--export-pdf={}'.format(result), tmpfile]
+        # e.g Linux: inkscape
+        cmd = ['inkscape', '--export-type=pdf', f'--export-filename={result}', tmpfile]

```

Comment:
Please don't erase the MAC path. Instead, using a built-in python module, detect the Operating System and in accordance to the answer, set a specific path. For example:

```
import os

if os.name == "nt":
    path = C:/Program Files/Inkscape/inkscape.exe
    cmd = [path, "...."]
elif os.name == "posix":
    path = cmd = ["/Applications/Inkscape.app/Contents/MacOS/inkscape", '--export-pdf={}'.format(result), tmpfile]
    # cmd = [path, "...."]
# same validation for Linux
```

Also, check if another module like "platform" identifies better the different OS names https://docs.python.org/3/library/os.html#os.name

Reply:
Fork the original project that already includes all these.

Reply:
Alright, let's improve it then. We can make python detect the OS and choose a path.
My goal with the proposal is to remove the manual editing :sweat_smile:

Reply:
And, for the record, you can take your time. We will use this after the event (probably in December).

Reply:
The initial PR was only to get it to work as is and organize it and document it all. It's done. You could add an issue or reject the PR. I'm done with this one. 😅